buffer/src/test/java/io/netty/buffer/ByteBufUtilTest.java

```diff
@@ -286,6 +286,56 @@ public void writeMediumBE(BufferType bufferType) {
         buf.release();
     }
 
+    @SuppressWarnings("deprecation")
+    @ParameterizedTest(name = PARAMETERIZED_NAME)
+    @MethodSource("noUnsafe")
+    public void readUnsignedShortBE(BufferType bufferType) {
+        int shortValue = 0x1234;
```

Comment:
Would be good to also test with a value that would be negative if the short was signed, to make sure there are no overflow bugs.

Reply:
Yes, should have done that :) Added a test and fixed one overflow issue